src/pages/georeference/control/controlClass.js

```diff
@@ -1,13 +1,15 @@
-import { render } from 'react-dom';
-
+import { createRoot } from 'react-dom/client';
 import Control from './control';
 
 class GeoreferenceControl {
   onAdd(map) {
     this.map = map;
     this.container = document.createElement('div');
     this.container.classList.add('azure-maps-control-container');
-    render(<Control map={this.map} />, this.container);
+
+    const root = createRoot(this.container);
+
+    root.render(<Control map={this.map} />);
```

Comment:
just curious whats the difference here?

Reply:
It is the React 18 syntax, previous was React 17, which was generating warnings in the console